example/src/App.js

```diff
@@ -45,7 +45,8 @@ const App = () => {
           disabledFeatures: disabledFeatures || '', // Will remove feature(s) from application. E.g. settings remoteid
           defaultDial: '',          // (strict-mode only) the default destination. E.g. 1234567890
           sessionsLimit: 3,         // limits amount of sessions user can have active
-          attendedTransferLimit: 2  // limits amount of attendedTransfer sessions user can have active
+          attendedTransferLimit: 2, // limits amount of attendedTransfer sessions user can have active
+          autoAnswer: true         // enable the auto-answer on incoming calls
```

Comment:
I definitely appreciate that you added the prop to the example, however my preference would be for this to be false by default.

Reply:
I got distracted, also wanted it to false for default, thanks for the help